interpreter/php/php_test.go

```diff
@@ -10,13 +10,17 @@ import (
 )
 
 func TestPHPRegexs(t *testing.T) {
-	shouldMatch := []string{"php", "./php", "/foo/bar/php", "./foo/bar/php", "php-fpm", "php-cgi7"}
+	shouldMatch := []string{"php", "./php", "/foo/bar/php", "./foo/bar/php", "php-fpm", "php-cgi7",
+		"/usr/lib/apache2/modules/libphp.so", "/libphp.so",
+		"/usr/lib/apache2/modules/libphp5.so", "/libphp5.so",
+		"/usr/lib/apache2/modules/libphp8.1.so", "/libphp8.1.so"}
```

Comment:
Just in case... Should we also include libphp.so.*? Like in this libz example (sorry, don't have libphp installed):
```
lrwxrwxrwx 1 root root     13 Oct 29  2024 /usr/lib/x86_64-linux-gnu/libz.so -> libz.so.1.3.1
lrwxrwxrwx 1 root root     13 Oct 29  2024 /usr/lib/x86_64-linux-gnu/libz.so.1 -> libz.so.1.3.1
-rw-r--r-- 1 root root 125376 Oct 29  2024 /usr/lib/x86_64-linux-gnu/libz.so.1.3.1
```

Reply:
I don't think so. Seems the apache module naming is quite specific, and does not follow the generic pattern on system libraries.

Reply:
From what I've see so far I tend to agree with @fabled. The libphp library is versioned in a specific way (by PHP release) and does not use version number suffix that is incremented whenever the interface changes.